WordPress/Classes/Services/MenusService.h

```diff
@@ -72,16 +72,12 @@ typedef void(^MenusServiceFailureBlock)(NSError *error);
            failure:(nullable MenusServiceFailureBlock)failure;
 
 /**
- *  @brief      Generate a list MenuItems from the blog's top-level pages.
+ *  @brief      Create a list MenuItems from the given page.
  *
- *  @param      blog      The blog to use for pages.  Cannot be nil.
- *  @param      success   The success handler.  Can be nil.
- *  @param      failure   The failure handler.  Can be nil.
+ *  @return     A MenuItem instance for the page if it's a top-level page. Otherwise, nil.
```

Comment:
Nitpick. There docs in the previous method has `@param`s. The new one doesn't. One could see this as a regression.

At the same time, I personally prefer this style of documentation, which I interpret as "only document was is not obvious."

What do you think? Is it worth using this as an example to start a team-wide conversation to define a standard?

Reply:
> "only document was is not obvious."

This is exactly what I was thinking. I don't think those param docs in the original function is really useful.